app/[locale]/profile/tabs.tsx

```diff
@@ -0,0 +1,96 @@
+          height="100"
+          width="100"
+          src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSkN_x7EwY4AART4pmSrWPhVDuaYdgOJjPjlH3bLUcegQ&s"
+        />
+        <h6 className="text-lg">12112232</h6>
+        <h6 className="mb-1 text-lg font-semibold text-primary-700">
+          Harsh Chauhan
+        </h6>
+      </div>
+      <nav className="mt-4 flex flex-col gap-1 space-y-1 *:scale-[0.85] *:rounded-xl *:border-[1px] *:border-primary-500 *:px-4 *:py-4 *:font-bold *:text-primary-700">
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Personal Details' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => setActiveTab('Personal Details')}
+        >
+          <MdOutlinePersonOutline size="1.75rem" />
+          <span>PERSONAL DETAILS</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Notifications' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => {
+            setActiveTab('Notifications');
+          }}
+        >
+          <IoMdNotificationsOutline size="1.75rem" />
+          <span>NOTIFICATIONS</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Courses' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => {
+            setActiveTab('Courses');
+          }}
+        >
+          <MdOutlineLibraryBooks size="1.75rem" />
+          <span>COURSES</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Clubs' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => {
+            setActiveTab('Clubs');
+          }}
+        >
+          <MdOutlineGroups size="1.75rem" />
+
+          <span>CLUBS</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Results and DMCs' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => setActiveTab('Results and DMCs')}
+        >
+          <LiaBookSolid size="1.75rem" />
+
+          <span>RESULTS AND DMCS</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Bookmarks' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => setActiveTab('Bookmarks')}
+        >
+          <MdOutlineBookmarks size="1.75rem" />
+
+          <span>BOOKMARKS</span>
+        </button>
+        <button
+          className={`flex items-center space-x-3 ${activeTab === 'Quick Send' && 'bg-primary-700 shadow-md *:text-background'}`}
+          onClick={() => setActiveTab('Quick Send')}
+        >
+          <TbFileSearch size="1.75rem" />
+
+          <span>QUICK SEND</span>
+        </button>
+      </nav>
+      <button className="mt-5 justify-center text-primary-700">LOG OUT</button>
```

Comment:
See this for logout:
https://github.com/nkss-dev/node-modules/blob/1d88b5974cd688caa7ca8cc5ba5047c5e6db109e/app/breadcrumb.tsx#L181

Reply:
pushed.